templates/default.html

```diff
@@ -2,10 +2,16 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Resin.io Documentation</title>
+    <title>{{ title }} - Resin.io Documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" type="image/x-icon" href="/favicon.png" />
 
+    <meta property="fb:app_id" content="221218511385682" />
```

Comment:
Hm, good question, habits, I guess? Thinking about it, it is a holdover from when I was contributing to Buildroot a bit, or was it some other project? Either way, most other project I've seen that if the PR needs changes, it comes in as another PR, not changes to the existing one (which is closed without merging) - ie. don't add commits to change things in PRs that needs fixing, and don't force push changes to existing commits.

I'm happy to do in any pattern that we use, haven't seen such info yet.